drone/client.go

```diff
@@ -70,6 +70,10 @@ const (
 	pathTemplates         = "%s/api/templates"
 	pathTemplateName      = "%s/api/templates/%s/%s"
 	pathTemplateNamespace = "%s/api/templates/%s"
+	pathCardsByBuild      = "%s/api/repos/%s/%s/cards/%v"
+	pathCreateCard        = "%s/rpc/v2/cards?stepNumber=%v"
```

Comment:
because the `/rpc/v2` endpoints are internal-use-only they are not including in our public sdk.  This should use the public path `%s/api/repos/%s/%s/cards/%v/%v/%v` and then the `/rpc/v2` endpoint would be added to `runner-go` instead, in the `client` package.